src/utils/viewer/viewer.cpp

```diff
@@ -639,57 +548,48 @@ void Viewer::mousePressEvent(MouseEvent& event) {
     createPickedObjectVisualizer(pickedObject);
     return;
   }  // drawable selection
+  // add primitive w/ right click
+  else if (event.button() == MouseEvent::Button::Right) {
+    if (simulator_->getPhysicsSimulationLibrary() !=
+        esp::physics::PhysicsManager::PhysicsSimulationLibrary::NONE) {
+      auto viewportPoint = event.position();
+      auto ray = renderCamera_->unproject(viewportPoint);
+      Corrade::Utility::Debug()
+          << "Ray: (org=" << ray.origin << ", dir=" << ray.direction << ")";
+
+      esp::physics::RaycastResults raycastResults = simulator_->castRay(ray);
+
+      for (auto& hit : raycastResults.hits) {
+        Corrade::Utility::Debug() << "Hit: ";
+        Corrade::Utility::Debug() << "  distance: " << hit.rayDistance;
+        Corrade::Utility::Debug() << "  object: " << hit.objectId;
+        Corrade::Utility::Debug() << "  point: " << hit.point;
+        Corrade::Utility::Debug() << "  normal: " << hit.normal;
+      }
```

Comment:
Alternative ideas: 

1. print this via imgui the same as you do with fps? (I'm not sure what's the intended use case -- if it's meant to be selected in the terminal then that's useless)
2. enable this printing only if some `--debug-raycast` option is enabled on the command-line? (would help with console noise if there's a lot of output, but probably makes no sense if all you need when clicking is seeing this info)

Reply:
Just removed this output. Originally meant for debugging anyway.